testsuite/go.mod

```diff
@@ -7,12 +7,11 @@ replace storj.io/gateway => ../
 require (
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
 	github.com/minio/minio v0.0.0-20201125204248-91130e884b5d
-	github.com/onsi/ginkgo v1.14.0 // indirect
 	github.com/stretchr/testify v1.6.1
 	github.com/zeebo/errs v1.2.2
 	go.uber.org/zap v1.16.0
-	storj.io/common v0.0.0-20210113135631-07a5dc68dc1c
+	storj.io/common v0.0.0-20210223102301-4de910503c2c
 	storj.io/gateway v0.0.0-00010101000000-000000000000
-	storj.io/storj v0.12.1-0.20210114091245-85fb964afefc
-	storj.io/uplink v1.4.5
+	storj.io/storj v0.12.1-0.20210223082219-0178ec77717f
+	storj.io/uplink v1.4.6-0.20210219140335-667f093f4a3a
```

Comment:
This will start using an unreleased version of uplink.

Reply:
Yeah, unfortunately have to do this for now.